### PR TITLE
tag view for git repository

### PR DESCRIPTION
**Pull request by rahulrrixe**

### CODE REVIEW

File: fresque/gitview.py [hunk 1]
```diff
@@ -219,3 +219,29 @@ def view_raw_file(repo, identifier, filename):
         headers['Content-Encoding'] = encoding
 
     return (data, 200, headers)
+
+
+@APP.route('/git/<repo>/tags/')
+@APP.route('/git/<repo>/tags')
+def view_tags(repo):
```
Comment (pypingou) [inline fresque/gitview.py]: Tags, where will those be used?

Comment (rahulrrixe) [inline fresque/gitview.py]: @pypingou it will used for marking the release point.

Comment (pypingou) [inline fresque/gitview.py]: on what? :)

Comment (rahulrrixe) [inline fresque/gitview.py]: @pypingou once the reviews for rpms gets closed and if contributor finds it ready for release for then he can tag it for release. Is it wrong?

Comment (pypingou) [inline fresque/gitview.py]: Do you see it in the present repos?

Comment (rahulrrixe) [inline fresque/gitview.py]: @pypingou I haven't seen in any of fedora git's repository. I think its not needed but still I don't know why?

Comment (pypingou) [inline fresque/gitview.py]: I sent you that link a while back: http://pkgs.fedoraproject.org/cgit/

**Comment by rahulrrixe** [comment]
This view is not needed.

**Pull request closed without merging**